engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -458,6 +461,18 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
     }
   }
 
+  private def saveSuccessfulJobResults(jobKey: JobKey, returnCode: Option[Int], outputs: JobOutputs) = {
+    val jobStoreKey = jobKey.toJobStoreKey(workflowId)
+    val jobStoreResult = JobResultSuccess(returnCode, outputs)
+    serviceRegistryActor ! RegisterJobCompleted(jobStoreKey, jobStoreResult)
```

Comment:
Yeah I did that at one point but I found the line to be a bit too wide and busy for my tastes.  😄